.github/workflows/update-api-schema.yml

```diff
@@ -57,10 +57,12 @@ jobs:
     - name: Create api schema dump
       run: |
         ./backend.ai mgr api dump-gql-schema --output docs/manager/graphql-reference/schema.graphql
+        ./backend.ai mgr api dump-gql-schema --v2 --output docs/manager/graphql-reference/strawberry-schema.graphql
         ./backend.ai mgr api dump-openapi --output docs/manager/rest-reference/openapi.json
     - name: Make commit message for changing change log file
       run: |
         git add docs/manager/graphql-reference/schema.graphql
+        git add docs/manager/graphql-reference/strawberry-schema.graphql
```

Comment:
Would `schema-v2.graphql` be better than `strawberry-schema.graphql`? @HyeockJinKim

Reply:
We have decided on the name "v2-schema.graphql."